### DISCUSSION TITLE
Nach Update auf v3.0.11 Fehler (Docker Port)

### DISCUSSION THREAD

**Discussion by bubiman** [Zefau/ioBroker.jarvis#1628] [Q&A] [answered]
Hallo,
anscheinend habe ich die Jarvis Instanz bei meinem iobroker die Tage aktualisiert und jetzt zeigt er nichts mehr an.
Die Instanz selber sieht gut aus
![image](https://user-images.githubusercontent.com/6061436/154969077-bd14cd9a-e286-4945-8350-8c748c8b3061.png)
aber die Seite dann selber leider nicht mehr
![image](https://user-images.githubusercontent.com/6061436/154969216-b431208a-8cb2-4cec-8e2b-6b52094756f0.png)
Die Konfiguration sieht wie folgt aus
![image](https://user-images.githubusercontent.com/6061436/154969582-a05de6a1-7b7f-4952-9372-1f4ad984eba6.png)

Kann mir jemand weiterhelfen?

**Comment by mcuiobroker** [comment]
F12 auf der Seite. Welche Meldungen sind zu sehen?

**Reply by bubiman** [reply]
> F12 auf der Seite. Welche Meldungen sind zu sehen?

![image](https://user-images.githubusercontent.com/6061436/154995840-03be7f00-3daf-4f0c-a610-d9aca7851297.png)
 Ist das das Richtige?

**Reply by mcuiobroker** [reply]
Welche web-Instanz web.0? Für jarvis zwingend notwendig web.0.
Web-Instanz-Einstellungen? auf eine IP?
Cache ?
- Browser Cache löschen
- WEB-Instanz neu starten
- ioBroker neu starten

**Reply by bubiman** [reply]
> Welche web-Instanz web.0? Für jarvis zwingend notwendig web.0. Web-Instanz-Einstellungen? auf eine IP? Cache ?
Ja web.0, AFAIK habe ich den Port geändert, da ein anderer Docker schon :8080 genutzt hatte:
![image](https://user-images.githubusercontent.com/6061436/155014555-e60966c0-2b0d-4b31-9bfb-5b27479bb614.png)

> * Browser Cache löschen

Gemacht und hat leider nichts gebracht. Habe auch mal nen anderen Browser (Firefox) versucht aber da kommt der gleiche Fehler

> * WEB-Instanz neu starten
> * ioBroker neu starten

Ja beides schon einmal neu gestartet. Habe gerade auch den ganzen Docker geupdatet und das Problem besteht noch.

**Comment by Zefau** [comment, marked as answer]
Was kommt, wenn du die URL direkt im Browser aufrufst?

**Reply by bubiman** [reply]
Die URL aus der Fehlermeldung des Browsers?
![image](https://user-images.githubusercontent.com/6061436/155015267-06f7ba81-a9bb-426f-be68-95acde22b904.png)

**Reply by Zefau** [reply]
Wenn du im Docker bist musst du ggf. den Port 8400 ins Netzwerk mappen?

**Reply by Zefau** [reply]
![image](https://user-images.githubusercontent.com/14368811/155015731-00e7977f-720a-4fb6-b99b-621620014600.png)

**Reply by bubiman** [reply]
Oh das ist mir jetzt peinlich. Ja das scheint das Problem gewesen zu sein.
Vielen dank euch.

**Comment by Bernd9000** [comment]
Kann das überhaupt funktionieren ?
Über Browser wurde https aufgerufen, web Instanz ohne ?

![Unbenannt](https://user-images.githubusercontent.com/77005685/155301469-c2cf6ea0-af34-4614-bc0a-0db46322ad2b.jpg)
![Unbenannt2](https://user-images.githubusercontent.com/77005685/155301478-22809acf-48eb-48a4-ab46-b416c6525710.jpg)
